snaps_k8s/provision/kubernetes/plugin/k8_impl/k8_utils.py

```diff
@@ -866,98 +860,44 @@ def remove_macvlan_networks(config, macvlan_master_hostname):
                     for item2 in multus_network:
                         for key2 in item2:
                             if key2 == "CNI_Configuration":
-                                cni_configuration = item2.get(
+                                cni_conf = item2.get(
```

Comment:
This looks almost identical to code in another file.

We should also have this separated into individual files for each CNI.

Reply:
Yes, there are several instances of this that I decided not to address with this PR.